tests/test_integration_service.py

```diff
@@ -44,7 +44,7 @@ def test_organisation(self):
             'user_key': 'AU',
             'unit_count': 1,
             'person_count': 2,
-            'employment_count': 3,
+            'employment_count': 4,
```

Comment:
This is technically wrong; we'll need to filter by `funktionsnavn` or something — or perhaps expand on this to list other types as well.

Reply:
I see. I'll merge this for now, and fix the test properly when I do writing for manager.